ha/core/health_monitor/monitor_rules_manager.py

```diff
@@ -105,9 +104,10 @@ def evaluate(self, event: HealthEvent) -> list:
             list: actions configured for the rule
         """
         val = []
-        if event.specific_info and event.specific_info.get(SPECIFIC_INFO_ATTRIBUTES.FUNCTIONAL_TYPE):
-            key = self._prepare_key(event.resource_type, event.event_type,
-                                    event.specific_info.get(SPECIFIC_INFO_ATTRIBUTES.FUNCTIONAL_TYPE))
+        if event.specific_info and event.specific_info.get(SPECIFIC_INFO_ATTRIBUTES.FUNCTIONAL_TYPE.value):
```

Comment:
Please check whether this requires f-string?

Reply:
SPECIFIC_INFO_ATTRIBUTES.FUNCTIONAL_TYPE.value returns value in string type only.
f-string is not required here.

Reply:
Ok..If you have verified. Then I am good.